docs/user/data_collection.rst

```diff
@@ -83,3 +83,106 @@ reported under the key ``shield-recipe-client/recipe/<recipe id>``:
+
+
+Enrollment
+-----------
+Shield records enrollment and unenrollment of users into studies, and
+records that data using `Telemetry Events`_. All data is stored in the
+``normandy`` category.
+
+.. _Telemetry Events: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html
+
+Preference Studies
+^^^^^^^^^^^^^^^^^^
+Enrollment
+   method
+      The string ``"enroll"``
+   object
+      The string ``"preference-study"``
+   value
+      The name of the study (``recipe.arguments.slug``).
+   extra
+      branch
+         The name of the branch the user was assigned to (example:
+         ``"control"`` or ``"experiment"``).
+      experimentType
+         The type of preference experiment. Currently this can take
+         values "exp" and "exp-highpop", the latter being for
+         experiments targetting large numbers of users.
+
+Unenrollment
+   method
+      The string ``"unenroll"``.
+   object
+      The string ``"preference-study"``.
+   value
+      The name of the study (``recipe.arguments.slug``).
+   extra
+      resetValue
+         The string ``"true"`` if the preference was set back to its
+         original value, ``"false"`` if it was left as its current
+         value. This can happen when, for example, the user changes a
+         preference that was involved in a user-branch study.
+      reason
+         The reason for unenrollment. Possible values are:
+
+         * ``"recipe-not-seen"``: The recipe was no longer
+           applicable to this client This can be because the recipe
+           was disabled, or the user no longer matches the recipe's
+           filter.
+         * ``"user-preference-changed"``: The study preference was
+           changed on the user branch. This ocould mean the user
+           changed the preference, or that some other mechanism set a
+           non-default value for the preference.
+         * ``"user-preference-changed-sideload"``: The study
+           preference was changed on the user branch while Shield was
+           inactive. This could mean that the value was manually
+           changed in a profile while Firefox was not running.
+
+Add-on Studies
+^^^^^^^^^^^^^^
+Enrollment
+   method
+      The string ``"enroll"``
+   object
+      The string ``"addon-study"``
+   value
+      The name of the study (``recipe.arguments.slug``).
+   extra
+      addonId
+         The add-on's ID (example: ``"feature-study@shield.mozilla.com"``).
+      addonVersion
+         The add-on's ID (example: ``"1.2.3"``).
+
+Unenrollment
+   method
+      The string ``"unenroll"``.
+   object
+      The string ``"addon-study"``.
+   value
+      The name of the study (``recipe.arguments.name``).
+   extra
+      addonId
+         The add-on's ID (example: ``"feature-study@shield.mozilla.com"``).
+      addonVersion
+         The add-on's ID (example: ``"1.2.3"``).
+      reason
+         The reason for unenrollment. Possible values are:
```

Comment:
From the code, looks like this could also be `"unknown"` ? But that value is not listed here...